lib/install/management/appliance.go

```diff
@@ -359,7 +359,7 @@ func (d *Dispatcher) configIso(conf *config.VirtualContainerHostConfigSpec, vm *
 		log.Errorf("Failed to create Cdrom device for appliance: %s", err)
 		return nil, err
 	}
-	cdrom = devices.InsertIso(cdrom, fmt.Sprintf("[%s] %s/appliance.iso", conf.ImageStores[0].Host, d.vmPathName))
+	cdrom = devices.InsertIso(cdrom, fmt.Sprintf("[%s] %s/%s", conf.ImageStores[0].Host, d.vmPathName, settings.ApplianceISO))
```

Comment:
Should we be using the actual path of the appliance VM rather than composing in this fashion. It'll currently work because we use ImageStores[0] as the applianceVM datastore, but if that ever changes then this will break.